youtube_dl/extractor/audiomack.py

```diff
@@ -9,20 +9,21 @@
 
 
 class AudiomackIE(InfoExtractor):
-    _VALID_URL = r'https?://(?:www\.)?audiomack\.com/song/(?P<id>[\w/-]+)'
+    _VALID_URL = r'https?://(?:www\.)?audiomack\.com/(song)/(?P<id>[\w/-]+)'
```

Comment:
Why is this change correct? This looks really strange and just adds a useless group, doesn't it?